PilotDataPlatform/metadata
# PILOT-1591 Add favourites data to items, collections returns

## Summary

- Adds a `favourite` field to the return from the items and collections search endpoints.
	- `/v1/items/search/` returns favourites data based on a new, optional `fav_user` param.
	- `/v1/collection/search/` always returns favourites data based on the existing `owner` param.

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1591

## Type of Change


## Testing


## Test Directions

Test the items and collections search endpoints to see if the `favourite` field in the return is accurate.
